src/akamai/js/akamaiHD_flash.js

```diff
@@ -1162,7 +1164,7 @@ var swfobject = function() {
       }
       if (ua.ie && ua.win) { // Internet Explorer + the HTML object element + W3C DOM methods do not combine: fall back to outerHTML
         var att = "";
-        for (var i in attObj) {
+        for (var i = 0; i < attObj.length; i++) {
```

Comment:
Fixed to revert back to original swfobject code on this. Updated code not reflecting here, but shows in 'files changed' tab.